src/transformers/models/transfo_xl/modeling_transfo_xl_utilities.py

```diff
@@ -109,7 +110,13 @@ def forward(self, hidden, labels=None, keep_order=False):
         if self.n_clusters == 0:
             logit = self._compute_logit(hidden, self.out_layers[0].weight, self.out_layers[0].bias, self.out_projs[0])
             if labels is not None:
-                out = -nn.functional.log_softmax(logit, dim=-1).gather(1, labels.unsqueeze(1)).squeeze(1)
+                if padding_index is not None:
+                    mask = labels != padding_index
```

Comment:
We should probably better check here if:
```python
mask = lables != -100
```
to be more in line with other models no? 
What do you think @sgugger ?

Reply:
Yes, that would be more consistent with how other models/PyTorch loss functions ignore some labels. Though it doesn't work the examples where we encourage `labels=inputs["input_ids"]`